Source/Core/Core/IOS/USB/Emulated/Skylanders/Skylander.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright 2022 Dolphin Emulator Project
 // SPDX-License-Identifier: GPL-2.0-or-later
 
-#include "Core/IOS/USB/Emulated/Skylander.h"
+#include "Skylander.h"
```

Comment:
Should be the full path `Core/IOS/USB/Emulated/Skylanders/Skylander.h` rather than the relative path.